jupyter-pytorch-full/rockcraft.yaml

```diff
@@ -58,18 +56,15 @@ parts:
   kubectl:
     plugin: nil
     stage-snaps:
-      - kubectl/1.21/stable
+      - kubectl/1.24/stable
     organize:
       kubectl: bin/kubectl
     stage:
       - bin/kubectl
 
   nodejs:
-    plugin: dump
-    # This is a bit of a hack to install a deb from an external apt repository.
-    # Once the apt repository support lands in Rockcraft this should be replaced.
-    source: https://deb.nodesource.com/node_14.x/pool/main/n/nodejs/nodejs_14.21.3-deb-1nodesource1_amd64.deb
-    source-type: deb
+    plugin: nil
+    stage-packages: [nodejs]
```

Comment:
Careful here - the version of nodejs in the repos might not be the one  you're looking for.

Take a look at how I solved this using `package-repositories` in the original repo: https://github.com/canonical/jupyter-pytorch-full-rock/commit/ab0cc479fd03f63e20bcc83212e7dca92abc0b80